R/ArmadilloConnection.R

```diff
@@ -283,9 +285,7 @@ methods::setMethod(
   function(conn) {
     response <- httr::GET(
       handle = conn@handle,
-      url = conn@handle$url,
-      path = "/workspaces",
-      httr::add_headers("Accept" = "application/json")
+      path = "/workspaces"
```

Comment:
Haha, removed the url again. This is what we did a while ago, we thought it would break things. at the time we couldn't get the handle to work the right way. Now it works, cleans up real good.